contracts/pricefeeds/WBTCPriceFeed.sol

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "../vendor/@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+import "../IPriceFeed.sol";
+
+/**
+ * @title WBTC price feed
+ * @notice A custom price feed that calculates the price for WBTC / USD
+ * @author Compound
+ */
+contract WBTCPriceFeed is IPriceFeed {
+    /** Custom errors **/
+    error BadDecimals();
+    error InvalidInt256();
+
+    /// @notice Version of the price feed
+    uint public constant override version = 1;
+
+    /// @notice Description of the price feed
+    string public constant override description = "Custom price feed for WBTC / USD";
+
+    /// @notice Number of decimals for returned prices
+    uint8 public immutable override decimals;
+
+    /// @notice Chainlink WBTC / BTC price feed
+    address public immutable WBTC_TO_BTC_PRICE_FEED;
```

Comment:
This is a nit, but generally we've reserved `UPPER_CASE` for constants, not immutables.

The [style guide](https://docs.soliditylang.org/en/latest/style-guide.html#constants) that OZ referenced only specifies this for constants.

Reply:
Good point! I reverted the `UPPER_CASE` change since there isn't a recommended style guide for immutables.